plugins/user/membyuser

```diff
@@ -17,12 +26,11 @@ if [ "$1" = "autoconf" ]; then
 fi
 
 if [ "$1" = "config" ]; then
-	echo "graph_args --base 1000 -r --lower-limit 0"
+	echo "graph_args --base 1024"
 	echo "graph_title Memory usage, by user"
-	echo "graph_category memory"
+	echo "graph_category system"
```

Comment:
I think, `memory` is suitable here, since the plugins focuses on this property.

Reply:
That is fair, the other 'xyzbyuser' scripts seem to be in the 'system' category (iirc), but I'll roll it back.